securing_apps/topics/docker/quickstart-zip.adoc

```diff
@@ -0,0 +1,22 @@
+[[_docker_quickstart_zip]]
+
+=== Docker Registry Quickstart Zip
+
+WARNING: This installation method is meant to be an easy way to get a docker registry authenticating against a keycloak server.  It is intended for development purposes only and should never be used in a production or production-like environment.
+
+The zip file installation mechanism provides a quickstart for developers who want to understand how the keycloak server can interact with the docker registry.  In order to configure the quickstart:
+
+ 1. Begin by creating a client configuration.  At this point you won't have a docker registry - the quickstart will take care of that part.
+ 1. Choose the "Quickstart" option from the installation tab and download the .zip file
+ 1. Unzip the archive to the desired location, and open the directory.
+ 1. Start the docker registry with `docker-compose up`
+ 1. Configure the realm in which the docker client is configured to use HTTP/Basic authentication as the browser auth flow
```

Comment:
This part about HTTP Basic Authentication should be more explicit - i.e.:

1. In Admin Console realm configuration under *Authentication* / *Flows* tab click *New* to create a new Authentication flow. Give it some alias e.g. 'docker auth'. For *Top Level Flow Type* select 'generic'.
1. Click on *Add execution* and select a provider called 'HTTP Basic Authentication'.
1. Make sure the execution is set to *Required*.
1. Go to *Bindings* tab, and change 'Browser Flow' to just created 'docker auth'.